addons/spreadsheet/static/src/pivot/plugins/pivot_core_plugin.js

```diff
@@ -279,12 +283,32 @@ export class PivotCorePlugin extends OdooCorePlugin {
     /**
      * @param {string} id
      * @param {PivotRuntime} definition
+     *
+     * @returns {PivotDefinition}
+     */
+    _convertPivotDefinition(id, definition) {
```

Comment:
maybe rename the `definition` param to `runtimeDefinition` ? It would be clearer to grasp which kind of definition